vignettes/manuscript.Rmd

```diff
@@ -264,24 +264,30 @@ To account for unique aspects of each lake and characterize lake productivity, w
-2. Using R's `randomForest` package, we develop two random forest models.
+2. Using R's `randomForest` package, we develop two random forest models (All Variables and GIS only).
 3. Assess model performance for both the predicted chlorophyll *a* and for categorical trophic state classifications.  Trophic state was defined using the NLA chlorophyll *a* trophic state cut offs (Table \ref{tab:trophicStateTable}). 
 
 ##Measures of Model Performance and Variable Importance
 
 We assessed the performance of the random forest two ways.  First we compare the root mean square error and the adjusted R\textsuperscript{2} of the models.  Second, we examine the accuracy of the model when converted to trophic states classes (Table \ref{tab:trophicStateTable}).   We assess the classifcations via a confusion matrix. A confusion matrix shows agreement and disagreement in a tabular form with predicted values forming the columns of the matrix and observed values, the rows.  From this tabulated information we calculate the total accuracy (i.e. percent correctly predicted) and the kappa coefficient, which takes into account the error expected by chance alone (i.e. the off diagonal values of the matrix) [@cohen1960coefficient; @hubert1985comparing].   The kappa coefficient can range from -1 to 1 with 0 equalling the agreement expected by chance alone.  Values greater than 0 represent agreement greater than would be expected by chance, with values greater than 0.61 considered "substantial" agreement  [@landis1977measurement].  Negative values are rare and would indicate no agreement between the predicted and observed values. Additonally, random forest builds each tree on bootstrapped, random subsets of the original data, thus, a separate independent validation dataset is not required and random forest error estimates are expected to be unbiased [@breiman2001random]. 
 
-The random forest algorithm explicitly measures variable importance with two metrics: mean decrease in Gini and percent increase in mean squared error.  For each of these they measure the impact on the overall model when that particular variable is included and thus can be used to assess importance [@breiman2001random]. The Gini Index has been shown to have a bias that is less apparent than with percent increase in mean squared error [@strobl2007bias], thus, we use this metric to asses variable importance.  Lastly, partial depenedence plots provide a mecahnism to examine the partial relationship between individual variables and the response variable [@jones2015exploratory]. We examine these plots for the top variables as assigned by percent increase in mean squared error of for each the reduced models. 
+<!---are they data bootstrapped? I thought that a random subset of the data were generated (as mentioned above) for each tree (i.e., sampled without replacement)--->
```

Comment:
OK, mind blown, but just cause I'm a dolt.  The bootstrap has a property of selecting ~63% of a given sample, thus for every tree there will be ~37% available for the OOB.  The actual number does vary, but not signficantly.  Hence the bootstrap and the OOB.  Cool!